.github/workflows/release_droid_prepare_original_checksum.yml

```diff
@@ -7,41 +7,22 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-          with:
-            fetch-depth: 0
-      - name: "Install Lua"
-        uses: leafo/gh-actions-lua@v8
```

Comment:
Did you remove the Lua installation by intention? This might brake the build.

Reply:
That came with the PK 2 switch. Fixed.